### PR TITLE
Fix/587 exercise setups names

### PR DESCRIPTION
**Pull request by rodrigondec**
# Resume
- #587 

Some app/project names were not fully updated to the new names!

Some apps are:
- a1 ecommerce (former a5)

> this PR is similar to https://github.com/globocom/secDevLabs/pull/598. But there's more mentions to the former A5 name besides those from the Makefile.

### CODE REVIEW

**Comment by fguisso** [comment]
Look that this complements PR #598 , we accept this PR but will let it on hold until the other merge.

File: owasp-top10-2021-apps/a1/ecommerce-api/Makefile
Comment (henriporto) [inline owasp-top10-2021-apps/a1/ecommerce-api/Makefile:12]: unused variable 😛

**Review by henriporto — APPROVED** [review]
good job! 👍

**Pull request merged**
